NuGet.Services.BasicSearch.CloudService/ServiceDefinition.csdef

```diff
@@ -0,0 +1,27 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<ServiceDefinition name="NuGet.Services.BasicSearch.CloudService" xmlns="http://schemas.microsoft.com/ServiceHosting/2008/10/ServiceDefinition" schemaVersion="2015-04.2.6">
+  <WebRole name="NuGet.Services.BasicSearch" vmsize="ExtraLarge">
+    <Sites>
+      <Site name="Web">
+        <Bindings>
+          <Binding name="Endpoint1" endpointName="Endpoint1" />
+        </Bindings>
+      </Site>
+    </Sites>
+    <ConfigurationSettings>
+      <Setting name="Microsoft.WindowsAzure.Plugins.Diagnostics.ConnectionString" />
+      <Setting name="Storage.Primary" />
+      <Setting name="Search.IndexContainer" />
+      <Setting name="Search.DataContainer" />
+      <Setting name="Search.IndexRefresh" />
+      <Setting name="Search.RegistrationBaseAddress" />
+    </ConfigurationSettings>
+    <Endpoints>
+      <InputEndpoint name="Endpoint1" protocol="http" port="80" />
```

Comment:
Should we add an HTTPS binding as well? Would like to see search run over HTTPS where possible.